### PR TITLE
MAM: add methods to load previous page and to load most recent page

### PR DESCRIPTION
**Pull request by ramabit**

### CODE REVIEW

File: smack-experimental/src/main/java/org/jivesoftware/smackx/mam/MamManager.java [hunk 1]
```diff
@@ -483,9 +521,9 @@ public MamPrefsResult retrieveArchivingPreferences() throws NoResponseException,
      * @throws InterruptedException
      * @throws NotLoggedInException
      */
-    public MamPrefsResult updateArchivingPreferences(List<Jid> alwaysJids, List<Jid> neverJids, DefaultBehavior defaultBehavior)
-            throws NoResponseException, XMPPErrorException, NotConnectedException, InterruptedException,
-            NotLoggedInException {
+    public MamPrefsResult updateArchivingPreferences(List<Jid> alwaysJids, List<Jid> neverJids,
```
Comment (Flowdalic) [inline smack-experimental/src/main/java/org/jivesoftware/smackx/mam/MamManager.java]: That's a non change, right?

**Comment by Flowdalic** [comment]
Could you squash and remove the non change?

**Comment by ramabit** [comment]
@Flowdalic Yes, I'll do it, sorry for the delay

**Comment by ramabit** [comment]
@Flowdalic Fixed and squashed

**Comment by Flowdalic** [comment]
Thanks, cherry picked as https://github.com/igniterealtime/Smack/commit/f7fcff139c617fff7a6fddec2cf98121638db9bd

**Pull request closed without merging**